dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClasspathCheckerTest.java

```diff
@@ -85,8 +84,7 @@ public void testFindInvalidReferences_arrayCloneMethod() throws IOException, URI
   public void testFindInvalidReferences_constructorInAbstractClass()
       throws IOException, URISyntaxException {
     List<Path> paths = ImmutableList.of(absolutePathOfResource("testdata/guava-23.5-jre.jar"));
-    ClasspathChecker classpathChecker =
-        ClasspathChecker.create(false, paths, ImmutableSet.copyOf(paths));
+    ClasspathChecker classpathChecker = ClasspathChecker.create(paths, ImmutableSet.copyOf(paths));
```

Comment:
If `ClasspathChecker.create()` took an `Iterable<Path>` as its second parameter, then the callers wouldn't all have to do the `ImmutableSet.copyOf()`.

Reply:
Removed the unnecessary `copyOf`s